### ISSUE TITLE
Species: Kurish

### ISSUE THREAD

**Issue by rosebrallier** [trowl223/Compound_X#581]
Let's get them plizards y'all.
Pending completion of Project 9: Species Re-work.

**Comment by bleehu** [comment]
Please put some design goals and explicit out-of-scope docs in the design repo for what the Kurish should accomplish/add to the game. The design isn't blocked by species re-work; only the specific implementation.
